dd-java-agent/agent-logging/src/main/java/datadog/trace/logging/PrintStreamWrapper.java

```diff
@@ -0,0 +1,67 @@
+package datadog.trace.logging;
+
+import datadog.trace.api.Platform;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.nio.file.Path;
+
+public class PrintStreamWrapper extends PrintStream {
+  private static final int lineSeparatorLength = Platform.isWindows() ? 2 : 1;
```

Comment:
Our code-style is to use upper-case for constants
```suggestion
  private static final int LINE_SEPARATOR_LENGTH = Platform.isWindows() ? 2 : 1;
```